controlpanel/frontend/static/javascripts/modules/role-names.js

```diff
@@ -0,0 +1,56 @@
+moj.Modules.roleNames = {
+  id: "role_name",
+  selectId: "#role_name-select",
+  roleEndpointAttr: "data-role-endpoint",
+  formClass: ".appRoles",
+  autocompleteWrapperClass: ".autocomplete__wrapper",
+  roleFilters: {
+    airflow: new RegExp('^airflow_'),
+    webapp: new RegExp('^(alpha|dev)_app_'),
+  },
+  roles: null,
+
+  init() {
+    this.$selectField = $(this.selectId);
+    this.roleListEndpoint = this.$selectField.attr(this.roleEndpointAttr);
```

Comment:
This should be
```javascript
this.roleListEndpoint = this.selectField.dataset.roleEndpoint;
```
See [Using data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes)